src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -86,6 +93,7 @@ public void addTermRating(HttpServletRequest request) throws IOException {
     Long professorRating = null;
     Long workHours = null;
     Long difficulty = null;
+    String userId = new String();
```

Comment:
Is this different from String userId;

Reply:
It is not! I just realized we could do that hahaha